client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -92,6 +110,12 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 						/>
 					</CityZipRow>
 
+					<TextControl
+						label={ __( 'Email address', 'woocommerce-admin' ) }
```

Comment:
I saw you remove the `'woocommerce-admin'` text domain in #59876. Should we do the same here?

Reply:
I'm not sure that was intentional. 🤔 I'll take a look.

Reply:
From (shaky) memory the JS translation functions adhere to text domains strictly so we need to remove it in this case to pick up calypso translations. In PHP we have some fallbacks (I think) to handle this sort of code mixing.

Reply:
Got it! I'll remove here as ell.